core/win32/inject.c

```diff
@@ -1065,7 +1065,6 @@ inject_gencode_at_ldr(HANDLE phandle, char *dynamo_path, uint inject_location,
  * process, mode_switch_buf_sz is maximum size for switch code, and
```

Comment:
> Note that allocation of vmheap fails upon initializing dynamoRIO 64 on WoW64 processes. Thus, we need to pass -reachable_heap to avoid having to make this allocation.

This should be solved by changing the default `vmheap_size` to be much smaller than 8GB for x64 DR inside WOW64.

Reply:
This is done in my last push.